glean-core/android/src/main/java/mozilla/telemetry/glean/private/PingType.kt

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.telemetry.glean.private
+
+import android.util.Log
+import mozilla.telemetry.glean.Glean
+
+/**
+ * This implements the developer facing API for custom pings.
+ *
+ * Instances of this class type are automatically generated by the parsers at build time.
+ *
+ * The Ping API only exposes the [send] method, which schedules a ping for sending.
+ */
+data class PingType(
```

Comment:
We should eventually talk about this and decide if this and the Ping registry stay on the kotlin side or move into Rust (-> followup)

Reply:
Yes, definitely as a follow-up.  I think it will ultimately have to live on the Rust side, since the Rust code will need to be able to send pings on its own (in response to persisted data for example) and will need to know things like whether to send the `client_id`.  I just consider this a stub to help more tests pass for now.